hw/ip/rstmgr/doc/theory_of_operation.md

```diff
@@ -151,6 +151,12 @@ Parameter                   | Default       | Description
 
 * [Interface Tables](../../../top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson#interfaces)
 
+## Reset Indications for Alert Handler
+
+The alert handler needs to know the status of the various reset domains in the system so that the ping mechanism does not produce false alert indications.
```

Comment:
```suggestion
The alert handler needs to know the status of the various reset domains in the system to avoid false alert indications due to the ping mechanism.
```

Reply:
done